### PR TITLE
Add DM webui guide

### PR DESCRIPTION
**Pull request by sunzhaoyang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/deploy-a-dm-cluster-using-tiup-offline.md [hunk 1]
```diff
@@ -110,6 +110,9 @@ monitoring_servers:
 
 grafana_servers:
   - host: 172.19.0.101
+    config:
+      auth.anonymous.enabled: true
```
Comment (Ehco1996) [inline dm/deploy-a-dm-cluster-using-tiup-offline.md:114]: Have you tested that these configurations actually work?

File: dm/dm-webui-guide.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    grafana_servers:
+      - host: 10.0.1.14
+        # port: 3000
+        # deploy_dir: /tidb-deploy/grafana-3000
+        config:
+          auth.anonymous.enabled: true
+          security.allow_embedding: true
+    ```
+
+2. 若`grafana_servers`使用了非默认的 IP 和端口，则需要在 WebUI 的 Dashboard 界面填写正确的 IP 和端口。
```
Comment (Ehco1996) [inline dm/dm-webui-guide.md:42]: For a dm cluster that has been upgraded to the latest version, how do DM know what the default port is?

File: dm/dm-webui-guide.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+[security]
+allow_embedding = true
+```
+
+然后执行`tiup dm reload`使新的配置生效即可。
+
+## 任务列表
+
+任务列表用于管理所有的迁移任务。与 v6.0 以下版本不同，WebUI 首先使用了新的生命周期管理（dmctl 预计将在 v6.1 版本同步修改）。这将导致操作方式与之前版本有所不同，可能需要一些适应。
```
Comment (Ehco1996) [inline dm/dm-webui-guide.md:58]: It is recommended that this point is not included in the documentation, as the work is not yet fully completed

File: dm/dm-webui-guide.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+---
+title: 使用 WebUI 管理 DM 迁移任务
+summary: 学习如何使用 WebUI 来方便的管理数据迁移任务
+---
+
+# 使用 WebUI 管理 DM 迁移任务
+
+DM WebUI 是为了方便大量迁移任务的管理、简化操作步骤的目的开发，其主要特性包含运行状态监控，上游数据源配置，以及迁移任务管理等。DM WebUI 可以从任意 master 节点访问，其端口与 OpenAPI 保持一致，默认为 8261。访问地址示例：`http://{master_ip}:{master_port}/dashboard/`。
+
+![webui](/media/dm/dm-webui-preview-en.png)
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:10]: 能否简要介绍一下左侧导航各项对应的页面是什么，以及用途呢？
image

File: dm/dm-webui-guide.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+1. 向导方式。通过 WebUI 根据指引一步步填写所需信息进行任务创建，此种方式比较适合入门用户及日常使用。
+2. 配置文件。通过直接粘贴或编写 JSON 格式的任务配置文件进行创建，支持更多的参数调整，适合熟练的用户使用。
+
+### 任务详情
+
+在任务列表中，点击任务名称，将在右侧滑出详情页面。该页面展示了更加详细的任务状态信息，允许查看每一个子任务的运行情况。并支持查看此迁移任务当前完整的配置项信息。
+
+在 DM 中，迁移任务中的每一个子任务可能处于不同的阶段，即全量导出（dump） -> 全量导入（load） -> 增量同步（sync）。因此任务的当前阶段以子任务所处阶段的统计信息来展示，可以更加清楚的了解任务运行情况。
+
+## 上游配置
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:85]: 这里对应左侧导航中的“source”吗？
image

Comment (Ehco1996) [reply dm/dm-webui-guide.md:73]: yes

File: dm/dm-webui-guide.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+创建迁移任务之前，需要先行创建所需要同步的上游数据源信息。请注意：
+
+1. GTID。如果存在主从切换，请务必在上游 MySQL 开启 GTID，并在创建上游配置时将 GTID 设为 True，否则数据迁移任务将在主从切换时中断（AWS Aurora 除外）；
```
Comment (Ehco1996) [inline dm/dm-webui-guide.md:77]: GTID 的描述是不是应该放在 Source 里，不太需要在 WebUI 的页面写吧？

File: dm/dm-webui-guide.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+## 同步详情
+
+如果需要查看迁移任务中所配置的迁移规则的运行情况，同步详情页面支持根据任务、数据源、表库名称进行查询。查询结果中将包含上游表至下游表的对应信息，因此请慎重使用`*`等通配符，以防止查询结果过多导致页面反应迟缓。
+
+## 集群管理
+
+### 成员列表
+
+此页面展示 DM 集群中所有的 master 和 worker 节点，以及 worker 节点与 source 的绑定关系。支持对 master 和 worker 执行简单的 online/office 操作。
```
Comment (Ehco1996) [inline dm/dm-webui-guide.md:89]: office -> offline

File: dm/dm-webui-guide.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+创建迁移任务之前，需要先行创建所需要同步的上游数据源信息。请注意：
+
+1. GTID。如果存在主从切换，请务必在上游 MySQL 开启 GTID，并在创建上游配置时将 GTID 设为 True，否则数据迁移任务将在主从切换时中断（AWS Aurora 除外）；
+2. 停用。若某个上游数据库需要临时下线，可将其“停用”，但停用期间其他正在同步的 MySQL 实例不可执行 DDL 操作，否则停用的实例被启用后将无法正常同步。
+3. relay log。当多个迁移任务使用同一个上游时，可能对其造成额外压力。建议开启 relay log 可降低对上游的影响。
+
+## 同步详情
+
+如果需要查看迁移任务中所配置的迁移规则的运行情况，同步详情页面支持根据任务、数据源、表库名称进行查询。查询结果中将包含上游表至下游表的对应信息，因此请慎重使用`*`等通配符，以防止查询结果过多导致页面反应迟缓。
```
Comment (Ehco1996) [inline dm/dm-webui-guide.md:83]: > 因此请慎重使用`*`等通配符，以防止查询结果过多导致页面反应迟缓。

用正则并不会结果变的更多，因为这里是从所有的库表里过滤用户想要的，如果什么过滤条件都不传，默认会返回所有库表，这样才是结果最多的情况

**Review by Ehco1996 — COMMENTED** [review]
restLGTM

File: dm/dm-webui-guide.md (hunk 6 above)
Comment (sunzhaoyang) [reply dm/dm-webui-guide.md:77]: 不嫌多，就怕用户没看到

File: dm/dm-webui-guide.md (hunk 8 above)
Comment (sunzhaoyang) [inline dm/dm-webui-guide.md:83]:
```suggestion
如果需要查看迁移任务中所配置的迁移规则的运行情况，同步详情页面支持根据任务、数据源、表库名称进行查询。查询结果中将包含上游表至下游表的对应信息，因此请慎重使用`.*`等，以防止查询结果过多导致页面反应迟缓。
```

File: dm/dm-webui-guide.md (hunk 7 above)
Comment (sunzhaoyang) [inline dm/dm-webui-guide.md:89]:
```suggestion
此页面展示 DM 集群中所有的 master 和 worker 节点，以及 worker 节点与 source 的绑定关系。支持对 master 和 worker 执行简单的 online/offline 操作。
```

File: dm/dm-webui-guide.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+```
+openapi = true
+```
+
+## Dashboard
+
+Dashboard 是内嵌了 DM 的 Grafana Dashboard，包含两个视图`Standard`和`Professional`分别展示了不同角度的监控信息。
+
+如果要在 DM WebUI 中正确显示 Dashboard，首次部署必须确认以下两个步骤正确完成：
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:30]: 首次部署，是指部署 DM 吗？

File: dm/dm-webui-guide.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+
+### 任务详情
+
+在任务列表中，点击任务名称，将在右侧滑出详情页面。该页面展示了更加详细的任务状态信息，允许查看每一个子任务的运行情况。并支持查看此迁移任务当前完整的配置项信息。
+
+在 DM 中，迁移任务中的每一个子任务可能处于不同的阶段，即全量导出（dump） -> 全量导入（load） -> 增量同步（sync）。因此任务的当前阶段以子任务所处阶段的统计信息来展示，可以更加清楚的了解任务运行情况。
+
+## 上游配置
+
+创建迁移任务之前，需要先行创建所需要同步的上游数据源信息。请注意：
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:75]: 能否先介绍下**上游配置**这个页面能帮助用户完成什么任务呢？

File: dm/dm-webui-guide.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+> **注意：**
+>
+> - DM WebUI 当前为实验特性，尚不建议用于生产环境。
+> 
+> - DM WebUI 中 task 的生命周期有所改变，不建议与 dmctl 同时使用。
+
+## 特性开关
+
+若要使用 DM WebUI，需要在 master 配置中开启`openapi`配置：
```
Comment (Ehco1996) [inline dm/dm-webui-guide.md:20]: 这个地方也要加入通过 tiup 部署时候的配置选项 类似于这个 commit 里的描述

https://github.com/pingcap/docs-cn/pull/8612/commits/33cd3bca3a5b1f6baa259b353265a266e57ba863

File: dm/dm-webui-guide.md (hunk 9 above)
Comment (Ehco1996) [reply dm/dm-webui-guide.md:30]: 这里是 grafana 的配置，应该是首次部署 grafana 时需要加入如下配置项

**Review by Ehco1996 — COMMENTED** [review]
rest LGTM

File: dm/dm-webui-guide.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+```
+openapi = true
+```
+
```
Comment (sunzhaoyang) [inline dm/dm-webui-guide.md:25]:
```suggestion
如需开启 OpenAPI，并且是通过 tiup 部署的 DM 集群则需要在拓扑文件中添加如下配置：

```yaml
server_configs:
  master:
    openapi: true
```

```

File: dm/dm-webui-guide.md (hunk 10 above)
Comment (sunzhaoyang) [reply dm/dm-webui-guide.md:75]: 填写上游信息啊，这个一般都能理解的

**Review by Ehco1996 — APPROVED** [review]

File: dm/dm-webui-guide-v1.md [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+        openapi = true
+        ```
+
+    - 如果你的 DM 集群是通过 TiUP 部署的，在拓扑文件中添加如下配置：
+
+        ```yaml
+        server_configs:
+        master:
+            openapi: true
+        ```
```
Comment (sunzhaoyang) [inline dm/dm-webui-guide-v1.md:53]:
```suggestion
        ```yaml
        server_configs:
          master:
            openapi: true
        ```
```

**Review by sunzhaoyang — COMMENTED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

File: dm/dm-webui-guide.md [hunk 14]
```diff
@@ -0,0 +1,126 @@
...
+        ```
+
++ 首次部署 Grafana 时，已正确安装监控相关组件：`monitoring_servers` 和 `grafana_servers`。`grafana_servers` 须按如下进行配置：
+
+    ```
+    grafana_servers:
+      - host: 10.0.1.14
+        # port: 3000
+        # deploy_dir: /tidb-deploy/grafana-3000
+        config:       # 请确保 tiup dm -v 版本在 v1.9.0 以上
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:62]:
```suggestion
        config:       # 请确保执行 tiup dm -v 的 TiUP 版本在 v1.9.0 及以上
```

File: dm/dm-webui-guide.md [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+> - DM WebUI 当前为实验特性，不建议在生产环境中使用。
+> - DM WebUI 中 `task` 的生命周期有所改变，不建议与 dmctl 同时使用。
+
+DM WebUI 主要包含以下界面：
+
+- **Dashboard**：包含了 DM 中同步任务的主要监控信息和运行状态信息，帮助用户快速了解任务的整体运行状况，以及与延迟、性能相关的重要信息。
+- **数据迁移**
+    - **任务列表**：提供创建迁移任务的界面入口，并展示各迁移任务的详细信息。帮助用户监控、创建、删除、配置迁移任务。
+    - **上游配置**：用户在此页面配置同步任务的上游数据源信息。管理数据迁移环境中的上游配置，包含了，新增、删除上游配置、监控上游配置对应的同步任务状态、修改上游配置等相关的管理功能。
+    - **同步详情**：展示更加详细的任务状态信息。根据用户指定的筛选条件查看同步任务的具体配置和状态信息，包括，上下游配置信息，上下游数据库名称、表名称等
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:24]:
```suggestion
    - **同步详情**：展示更加详细的任务状态信息。根据用户指定的筛选条件查看同步任务的具体配置和状态信息，包括，上下游配置信息，上下游数据库名称、表名称等。
```

File: dm/dm-webui-guide.md [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+summary: 学习如何使用 WebUI 来方便的管理数据迁移任务。
+---
+
+# 使用 WebUI 管理 DM 迁移任务
+
+DM WebUI 是一个 TiDB Data Migration (DM) 迁移任务管理界面，方便用户以直观的方式管理大量迁移任务，无需使用 dmctl 命令行，简化任务管理的操作步骤。
+
+本文档介绍 DM WebUI 的访问方式、使用前提、各界面的使用场景以及注意事项。
+
+
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:12]:
```suggestion
```

File: dm/dm-webui-guide.md [hunk 17]
```diff
@@ -0,0 +1,126 @@
...
+> - DM WebUI 当前为实验特性，不建议在生产环境中使用。
+> - DM WebUI 中 `task` 的生命周期有所改变，不建议与 dmctl 同时使用。
+
+DM WebUI 主要包含以下界面：
+
+- **Dashboard**：包含了 DM 中同步任务的主要监控信息和运行状态信息，帮助用户快速了解任务的整体运行状况，以及与延迟、性能相关的重要信息。
+- **数据迁移**
+    - **任务列表**：提供创建迁移任务的界面入口，并展示各迁移任务的详细信息。帮助用户监控、创建、删除、配置迁移任务。
+    - **上游配置**：用户在此页面配置同步任务的上游数据源信息。管理数据迁移环境中的上游配置，包含了，新增、删除上游配置、监控上游配置对应的同步任务状态、修改上游配置等相关的管理功能。
+    - **同步详情**：展示更加详细的任务状态信息。根据用户指定的筛选条件查看同步任务的具体配置和状态信息，包括，上下游配置信息，上下游数据库名称、表名称等。
```
Comment (TomShawn) [inline dm/dm-webui-guide.md:23]:
```suggestion
    - **同步详情**：展示更加详细的任务状态信息。根据用户指定的筛选条件查看同步任务的具体配置和状态信息，包括上下游配置信息，上下游数据库名称，源表和目标表的关系等。
```

File: dm/dm-webui-guide.md [hunk 18]
```diff
@@ -0,0 +1,126 @@
...
+
+你可以通过**同步详情**页面查看迁移任务中所配置迁移规则的运行情况。同步详情页面支持根据任务、数据源、表库名称进行查询。
+
+查询结果中包含上游表至下游表的对应信息，因此请慎重使用 `.*` 等，以防止查询结果过多导致页面反应迟缓。
+
+## 集群管理
+
+### 成员列表
+
+**成员列表**页面展示 DM 集群中所有的 master 和 worker 节点，以及 worker 节点与 source 的绑定关系。你可在此页面对 master 和 worker 节点执行简单的 online 或 offline 操作。
```
Comment (sunzhaoyang) [inline dm/dm-webui-guide.md:126]:
```suggestion
**成员列表**页面展示 DM 集群中所有的 master 和 worker 节点，以及 worker 节点与 source 的绑定关系。
```

**Pull request merged**